src/safeAppsSdkConnector/index.ts

```diff
@@ -0,0 +1,77 @@
+import initSdk, {
+  RequestId,
+  SafeInfo,
+  SdkInstance,
+  TxConfirmationEvent,
+  TxRejectionEvent
+} from '@gnosis.pm/safe-apps-sdk'
+import { v4 as uuidv4 } from 'uuid'
+import SafeAppsSdkTransactionManager from '../transactionManagers/SafeAppsSdkTransactionManager'
+import { SafeAppsSdkTransactionResult, StandardSafeAppsTransaction } from '../utils/transactions'
+
+interface TxCallback {
+  confirm: (txResult: SafeAppsSdkTransactionResult) => void
+  reject: (error: Error) => void
+}
+
+class SafeAppsSdkConnector {
```

Comment:
Is this class used anywhere? I don't see it in the tests...

Reply:
In the CPK consctructor

Reply:
This is very similar to the transaction manager, as noted previously, and even supercedes the functionality of the manager when the number of transactions is not zero.